test/BUILD

```diff
@@ -128,7 +128,59 @@ load("//test:prism_location_test.bzl", "prism_location_test_suite")
 # Run the macro
 prism_location_test_suite(
     name = "prism_location_tests",
-    srcs = glob(["prism_regression/**/*.rb"]),
+    srcs = glob(
+        ["prism_regression/**/*.rb"],
+        exclude = [
```

Comment:
Printing the nodes' actual locations uncovered discrepancies in the Prism translation.

I have about 10 PRs that I'll open in the coming days which chip away at these, bit by bit.